README.rst

```diff
@@ -84,3 +89,10 @@ Targets:
 * pacman (ArchLinux) packages
 
 .. include: docs/contributing
+
+.. |Build| image:: https://img.shields.io/travis/jordansissel/fpm.svg
+   :target: https://travis-ci.org/jordansissel/fpm
+.. |Chat| image:: https://img.shields.io/gitter/room/jordansissel/fpm.svg
```

Comment:
http://fpm.readthedocs.io/en/latest/contributing.html recommends #fpm on freenode irc. If you desire folks relocate to gitter, then some investment needs to occur in moving folks from IRC to Gitter.